docker/addons/mongodb-reader/docker-compose.grpc-mtls.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+  mongodb-reader:
+    environment:
+      # Users gRPC client environmental varaibles
+      MF_AUTH_GRPC_CLIENT_CERT: /users-grpc-client.crt
+      MF_AUTH_GRPC_CLIENT_KEY: /users-grpc-client.key
+      MF_AUTH_GRPC_SERVER_CA_CERTS: /users-grpc-server-ca.crt
+      # Things gRPC client environmental varaibles
+      MF_THINGS_AUTH_GRPC_CLIENT_CERT: /client.crt
+      MF_THINGS_AUTH_GRPC_CLIENT_KEY: /client.key
+      MF_THINGS_AUTH_GRPC_SERVER_CA_CERTS: /server_ca.crt
+    volumes:
+      # Users gRPC client certificates
+      - ${MF_ADDONS_CERTS_PATH_PREFIX}${MF_USERS_GRPC_CLIENT_CERT}:/users-grpc-client.crt
```

Comment:
why two envs and then a constant suffix
wouldnt be easier to have only MF_USERS_GRPC_CLIENT_CERT to specify the full path to file

Reply:
Env Var `MF_USERS_GRPC_CLIENT_CERT` is commonly used by all the service 
From `docker/docker-compose.yml`, path in present in  `MF_USERS_GRPC_CLIENT_CERT` will be valid , but 
from `docker/addons/docker-compose.yml` path present in ` MF_USERS_GRPC_CLIENT_CERT` will be invalid. 
So make the path valid for `addons` a prefix var `MF_ADDONS_CERTS_PATH_PREFIX` is used